waku/mailserver.md

```diff
@@ -62,7 +62,7 @@ The `Cursor` field SHOULD be filled in if a number of envelopes between `Lower`
 
 The envelope MUST be encrypted with a symmetric key agreed between the requester and Mailserver.
 
-If `topic-interest` is used the `Cursor` field MUST be specified for the argument order to be unambiguous. However, it MAY be set to null. `topic-interest` is used to specify which topics a node is interested in. If this is specified, a mailserver MUST NOT send messages that aren't in in that topic. When `topic-interest` is set (even if it an empty array), the `bloom` filter parameter should be ignored.
+If `Topics` is used the `Cursor` field MUST be specified for the argument order to be unambiguous. However, it MAY be set to null. `Topics` is used to specify which topics a node is interested in. If `Topics` is not empty, a mailserver MUST only send messages that belong to a topic from `Topics` list and `Bloom` value MUST be ignored.
```

Comment:
```suggestion
If `Topics` is used the `Cursor` field MUST be specified for the argument order to be unambiguous. However, it MAY be set to `null`. `Topics` is used to specify which topics a node is interested in. If `Topics` is not empty, a mailserver MUST only send messages that belong to a topic from `Topics` list and `Bloom` value MUST be ignored.
```

Reply:
Also, it might make sense to completely reword this sentence to use set language.

Reply:
What's "set language"? Treat `Topics` as a collection?

Reply:
Treat topics as a set, eg "If topic is an element of Topics."